DEVELOP.rst

```diff
@@ -0,0 +1,42 @@
+Using the development buildout
+==============================
+
+Create a virtualenv in the package::
+
+    $ virtualenv --clear .
+
+Install requirements with pip::
+
+    $ ./bin/pip install -r requirements.txt
+
+Run buildout::
+
+    $ ./bin/buildout
+
+Start Plone in foreground:
+
+    $ ./bin/instance fg
+
+
+Running tests
+-------------
+
+    $ tox
+
+list all tox environments:
+
+    $ tox -l
```

Comment:
It seems to me this `tox -l` output is not the same we have for this package.

Reply:
I didn't quite understand it, but I saw that it has configurations of Plone versions not supported by this package. I'ill remove them.

Reply:
Ok, I see there is a mismatch between this list and what we have on travis as well